### ISSUE TITLE
MRQ_MONGODB_LOGS_SIZE seems to be non working

### ISSUE THREAD

**Issue by niksite** [pricingassistant/mrq#112]
Hi,

I have added the following environment value to make logs db capped:
    - MRQ_MONGODB_LOGS_SIZE=100000000

But I see the following in the db:

```
# mongo
...
> use mrq
switched to db mrq
> db.mrq_logs.stats()
{
    "ns" : "mrq.mrq_logs",
    "count" : 19755,
    "size" : 30191135,
    "avgObjSize" : 1528,
    "storageSize" : 8024064,
    "capped" : false,
    "wiredTiger" : {
...

```

So, it seems not to be capped yet.

**Comment by achauve** [comment]
Hi @sylvinus 
I've looked in the code and I can't find where the "MONGODB_LOGS_SIZE" is used either. Do you use it at PricingAssistant?

Thanks !

**Comment by sylvinus** [comment]
I can confirm this has unfortunately been broken for a while. The best way is to setup this collection by hand with the settings you want.

We want to add a pluggable log handler soon, so that MRQ can log to mongodb or any other backend.

**Comment by maelorn** [comment]
Fixed in 2815446
